Integrations/integration-Tenable_io.yml

```diff
@@ -0,0 +1,673 @@
+        if len(vulns_not_found) > 0:
+            vulns_not_found = replace_keys(vulns_not_found)
+            results.append(get_entry_for_object('Vulnerabilities - Missing From Workbench', 'TenableIO.Vulnerabilities', vulns_not_found, SCAN_REPORT_VULNERABILITIES_HEADERS, True))
+
+        if detailed == 'yes':
+            assets = replace_keys(scan_details['hosts'] + scan_details['comphosts'])
+            results.append(get_entry_for_object('Assets', 'TenableIO.Assets', assets, SCAN_REPORT_HOSTS_HEADERS))
+            if 'remediations' in scan_details and 'remediations' in scan_details['remediations'] and len(scan_details['remediations']['remediations']) > 0:
+                remediations = replace_keys(scan_details['remediations']['remediations'], REMEDIATIONS_NAMES_MAP)
+                results.append(get_entry_for_object('Remediations', 'TenableIO.Remediations', remediations, SCAN_REPORT_REMEDIATIONS_HEADERS))
+        return results
+
+    def get_vulnerability_details(plugin_id, date_range):
+        info = send_vuln_details_request(plugin_id, date_range)
+        if 'error' in info:
+            return { 'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': info['error'] }
+        return get_entry_for_object('Vulnerability details - {}'.format(plugin_id), 'TenableIO.Vulnerabilities', replace_keys(flatten(info['info'])), VULNERABILITY_DETAILS_HEADERS)
+
+    def get_vulnerabilities_by_asset(asset_id=None, hostname=None, ip=None, date_range=None):
+        if all(s is None for s in (asset_id, hostname, ip)):
+            return { 'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': "Please provide one of the following arguments: asset-id, hostname, ip" }
+        indicator = None
+        if asset_id is None:
+            asset_id, indicator, err = get_asset_id(hostname, ip)
+            if err:
+                return err
+        else:
+            indicator = asset_id
+        info = send_asset_vuln_request(asset_id, date_range)
+        if 'error' in info:
+            return { 'Type': entryTypes['error'], 'ContentsFormat': formats['text'], 'Contents': info['error'] }
+        vulns = replace_keys(info['vulnerabilities'], ASSET_VULNS_NAMES_MAP)
+        return get_entry_for_object('Vulnerabilities for asset {}'.format(indicator), 'TenableIO.Vulnerabilities', vulns, ASSET_VULNS_HEADERS)
+
+    def get_scan_status(scan_id):
+        scan_details = send_scan_request(scan_id)
+        scan_status = { 'Id': scan_id, 'Status': scan_details['info']['status'] }
+        return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan', scan_status)
+
+
+    # Command selector
+    if demisto.command() == 'test-module':
+        demisto.results(test_module())
+    elif demisto.command() == 'tenable-io-list-scans':
+        demisto.results(get_scans(demisto.getArg('folderId'), demisto.getArg('lastModificationDate')))
```

Comment:
the args we extract inside of the `_command` function and pass them explicitly to inner functions